hw/syn/data/dc.hjson

```diff
@@ -26,5 +26,8 @@
                 "--outdir {build_dir}"]
 
   // Restrict the maximum message count to 10 in each category
-  max_msg_count: 10
+  max_msg_count: 20
```

Comment:
you might want to change the comment in line 28 as well to `20`

Reply:
right, thanks!